test/core/rules/test_cases/L017.yml

```diff
@@ -0,0 +1,24 @@
+rule: L017
+
+passing_example:
+  pass_str: SELECT SUM(1)
+
+simple_fail:
+  fail_str: SELECT SUM (1)
+  fix_str: SELECT SUM(1)
+
+complex_fail_1:
```

Comment:
To make sure I understand, with these last two test cases, the rule is not changing the code, is that right? If so, these test cases look 💯.

Meta-comment: It'd be useful if there were a way for a test case to specifically state that "this is a failing case, but the linter should not change it". Maybe setting `fix_str` to `null`??